proto/v1/grafeas.proto

```diff
@@ -239,6 +240,10 @@ message Occurrence {
     grafeas.v1.UpgradeOccurrence upgrade = 15;
     // Describes a compliance violation on a linked resource.
     grafeas.v1.ComplianceOccurrence compliance = 16;
+    // https://github.com/secure-systems-lab/dsse
+    grafeas.v1.DsseAttestationOccurrence dsse_attestation = 17;
+    // https://github.com/secure-systems-lab/dsse
```

Comment:
`envelope` should be in a top-level of the Occurrence, not as a `oneof` field

Reply:
Ah, good catch, thanks.